### PR TITLE
Add tags and exclude-tags support

### LINKED ISSUE #492: Add support for test tags to the CLI
The built-in [test](https://pub.dev/packages/test) package lets developers add _tags_ to their test files, groups, and individual tests.

I won't ramble about tags here, instead I'll send you to the official docs:
- [Tagging tests](https://pub.dev/packages/test#tagging-tests) in `package:test`'s readme
- [Configuring tags](https://github.com/dart-lang/test/blob/master/pkgs/test/doc/configuration.md#configuring-tags) in `package:test`'s docs

We should research:
- if tags are good for marking platform-specific tests
- if there are any problems with them. If not, we should add support for running tests by tags to `patrol_cli`, and update docs to let our users know about this feature.

cc: @jakubfijalkowski @mateuszwojtczak @jBorkowska, wdyt?

### PR DESCRIPTION
**Pull request by pdenert**
fix #492

### CODE REVIEW

File: packages/patrol/lib/src/common.dart [hunk 1]
```diff
@@ -169,12 +170,15 @@ DartGroupEntry createDartTestGroup(
...
   int maxTestCaseLength = global_state.maxTestLength,
+  String? tags,
+  String? excludeTags,
 }) {
   final groupDTO = DartGroupEntry(
     name: name,
     type: GroupEntryType.group,
     entries: [],
     skip: parentGroup.metadata.skip,
+    tags: parentGroup.metadata.tags.toList(),
```
Comment (mateuszwojtczak) [inline packages/patrol/lib/src/common.dart:181]: what about `excludeTags` in here?

**Review by mateuszwojtczak — APPROVED** [review]
LGTM

File: packages/patrol/lib/src/common.dart (hunk 1 above)
Comment (mateuszwojtczak) [reply packages/patrol/lib/src/common.dart:181]: Ah, it doesn't make sense, right?

Comment (pdenert) [reply packages/patrol/lib/src/common.dart:181]: Yeah

**Review by piotruela — APPROVED** [review]
Nice 👌

**Pull request merged**